### PR TITLE
adding MAP_STRING_VECTOR_PAIR_INT_PAIR_STRING_STRING

### PR DESCRIPTION
**Pull request by gidden**
a ridiculously long container, but useful for a better growth region

### CODE REVIEW

**Comment by gidden** [comment]
2^5 == 32 possible perturbations of variable container lengths =(

File: tests/sqlite_back_tests.cc [hunk 1]
```diff
@@ -44,27 +44,6 @@ TEST_F(SqliteBackTests, MapStrDouble) {
   EXPECT_EQ(5.5, m["three"]);
 }
 
-TEST_F(SqliteBackTests, MapIntMapStringDouble) {
```
Comment (gonuke) [inline tests/sqlite_back_tests.cc]: Why are we removing this test?

**Comment by gidden** [comment]
image

File: tests/sqlite_back_tests.cc (hunk 1 above)
Comment (gidden) [inline tests/sqlite_back_tests.cc]: whoops!

**Comment by rwcarlsen** [comment]
Those permutations get out of hand quick!

**Pull request merged**